tasks/extract_filepaths_for_granule/test/unit_tests/test_extract_file_paths_for_granule.py

```diff
@@ -238,152 +238,16 @@ def test_task(self, mock_debug: MagicMock):
             extract_filepaths_for_granule.OUTPUT_DESTINATION_BUCKET_KEY: "sndbx-cumulus-public",
         }
         exp_gran = {
-            "dataType": "MOD09GQ_test-jk2-IngestGranuleSuccess-1558420117156",
```

Comment:
Since we are no longer copying the input granules to the output, several of these tests become much simpler.